### PR TITLE
#1162 Add the preassembled models of generators with impedance

### PR DESCRIPTION
**Pull request by cossartque**
This PR adds the preassembled models of the generators with impedance.

### CODE REVIEW

File: dynawo/sources/Models/Modelica/PreassembledModels/GeneratorPVDiagramPQSignalNWithImpedance.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, you can obtain one at http://mozilla.org/MPL/2.0/.
+    SPDX-License-Identifier: MPL-2.0
+
+    This file is part of Dynawo, an hybrid C++/Modelica open source suite of
+    simulation tools for power systems.
+-->
+<dyn:dynamicModelsArchitecture xmlns:dyn="http://www.rte-france.com/dynawo">
+  <dyn:modelicaModel id="GeneratorPVDiagramPQSignalNWithImpedance">
+    <dyn:unitDynamicModel id="coupling" name="Dynawo.Electrical.Switches.IdealSwitch"/>
```
Comment (MarianneSaugierRTE) [inline dynawo/sources/Models/Modelica/PreassembledModels/GeneratorPVDiagramPQSignalNWithImpedance.xml:16]: Do you need the coupling + the line?

File: dynawo/sources/Models/Modelica/PreassembledModels/GeneratorPVSignalNWithImpedance.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, you can obtain one at http://mozilla.org/MPL/2.0/.
+    SPDX-License-Identifier: MPL-2.0
+
+    This file is part of Dynawo, an hybrid C++/Modelica open source suite of
+    simulation tools for power systems.
+-->
+<dyn:dynamicModelsArchitecture xmlns:dyn="http://www.rte-france.com/dynawo">
+  <dyn:modelicaModel id="GeneratorPVSignalNWithImpedance">
+    <dyn:unitDynamicModel id="coupling" name="Dynawo.Electrical.Switches.IdealSwitch"/>
```
Comment (MarianneSaugierRTE) [inline dynawo/sources/Models/Modelica/PreassembledModels/GeneratorPVSignalNWithImpedance.xml:16]: same

File: dynawo/sources/Models/Modelica/PreassembledModels/CMakeLists.txt [hunk 3]
```diff
@@ -23,7 +23,9 @@ set(MODELICA_PREASSEMBLED_MODELS
     GeneratorPQPropSignalN.xml
     GeneratorPV.xml
     GeneratorPVSignalN.xml
+    GeneratorPVSignalNWithImpedance.xml
```
[the remark this answers is not in the document]
Comment (cossartque) [reply dynawo/sources/Models/Modelica/PreassembledModels/CMakeLists.txt:26]: Ok, done

File: dynawo/sources/Models/Modelica/PreassembledModels/GeneratorPVDiagramPQSignalNWithImpedance.xml (hunk 1 above)
Comment (cossartque) [reply dynawo/sources/Models/Modelica/PreassembledModels/GeneratorPVDiagramPQSignalNWithImpedance.xml:16]: I need the coupling to give coupling_PGenPu in the staticRef

**Review by marcochiaramello — APPROVED** [review]

**Review by MarianneSaugierRTE — APPROVED** [review]

**Pull request merged**